### ISSUE TITLE
Polygon MYST top-up takes very long to appear

### ISSUE THREAD

**Issue by Donatas-MN** [mysteriumnetwork/payments#240]
Steps to reproduce:
1. Use either Desktop or Mobile app to top-up Polygon MYST.
2. Send MYST to required channel address.
3. Notice that it takes very long for MYST to appear in https://hermes3.mysterium.network/api/v1/data/consumer/0xc603e2a1739eb9288eb4985e74acbd80fbd10b71 after successful transaction.

It also looks like that restarting app helps to get the balance.

Tasks:
 - [ ] investigate if it's payments infra or apps issue.

**Comment by Guillembonet** [comment]
Yes, it takes a bit, but node is able to refresh the balance using blockchain data and it works well. You should use the `/identities/{identity}/balance/refresh` endpoint to refresh it (note that it also has an internal cache of a minute, so it only loads once a minute if you call it repeatedly). For mobile the binding is: `ForceBalanceUpdate`.

Tested the following workflow:
1. Send MYST to required channel address.
2. Wait for tx to be mined
3. Call the mentioned endpoint
4. See balance instantly
